File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for height
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Print out hashes base on height
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            // Equation to print number of spaces first
            if ((i + j) < (n - 1))
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# Prompt user for height between 1 and 8
height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")

# Build pyramid
for i in range(height):
    for j in range(height):
        if (i + j) < (height - 1):
            print(" ", end="")
        else:
            print("#", end="")
    # Print out space between pyramids
    print("  ", end="")
    # Print out left-aligned hashes
    for k in range(i + 1):
        print("#", end="")
    print()
